readme.md

```diff
@@ -8,8 +8,7 @@ Thanks to everyone who continues to contribute; new languages are created every
 Make sure to see [contributing.md](/contributing.md) for instructions on contributing to the project!
 
 <!--Languages start-->
-
```

Comment:
Can you replace the line that was removed.

Reply:
What do you mean? You mean that I need to put line 10 on line 11?

Reply:
I've done it